penguin/cli/cli.py

```
@@ -739,6 +744,212 @@ async def _run_penguin_direct_prompt(prompt_text: str, output_format: str):
+        ))
+        console.print(f"[bold]Description:[/bold] {skill.description}")
+        if skill.allowed_tools:
+            console.print(f"[bold]Allowed tools hint:[/bold] {', '.join(skill.allowed_tools)}")
+
+    asyncio.run(_run())
+
+
+@skill_app.command("activate")
+def skill_activate(
+    name: str = typer.Argument(..., help="Skill name"),
+    json_output: bool = typer.Option(False, "--json", help="Output as JSON"),
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)